policies/requirements.md

```diff
@@ -0,0 +1,50 @@
+# Web of Things (WoT) Policy
+
+## Requirements 
+New WoT specification Features should be motivated by Use Cases, as when updated specifications
+are published we should be able to  specify exactly what problem each new feature
+(or change to an existing feature) is solving.
+
+The following requirements capture process is centered on User Stories, which link Use Cases to Features
```

Comment:
How exactly are Use Cases & User Stories meant to be used together? When should someone use one or the other and what is the relationship between the two?

Reply:
Also see https://github.com/w3c/wot-thing-description/issues/2098